test_outputs/cli/acs-security-demos_focus_workload_ingress-controller_connlist_output.txt

```diff
@@ -1,2 +1,4 @@
-{ingress-controller} => frontend/asset-cache[Deployment] : TCP 8080
-{ingress-controller} => frontend/webapp[Deployment] : TCP 8080
\ No newline at end of file
+{ingress-nginx} => frontend/asset-cache[Deployment] : TCP 8080
+{ingress-nginx} => frontend/webapp[Deployment] : TCP 8080
+{openshift-ingress-operator} => frontend/asset-cache[Deployment] : TCP 8080
+{openshift-ingress-operator} => frontend/webapp[Deployment] : TCP 8080
```

Comment:
I think it is confusing to see multiple options for ingress controllers in the report..

Reply:
will use general {ingress-controller} when both are enabled (updated doc, wip code)